packages/amazonq/.changes/next-release/Bug Fix-993929aa-5dd5-440a-9150-8ae5ba5f56ac.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Include mvn and gradle files in repo archives"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).

Reply:
updated